db/config.go

```diff
@@ -91,13 +92,13 @@ func WriteBoard(tx *sql.Tx, c BoardConfigs) error {
 			"id", "readOnly", "textOnly", "forcedAnon", "disableRobots",
 			"flags", "NSFW",
 			"rbText", "pyu", "created", "defaultCSS", "title",
-			"notice", "rules", "eightball",
+			"notice", "rules", "eightball", "cinemaEnabled",
 		).
 		Values(
 			c.ID, c.ReadOnly, c.TextOnly, c.ForcedAnon, c.DisableRobots,
 			c.Flags, c.NSFW, c.RbText, c.Pyu,
 			c.Created, c.DefaultCSS, c.Title, c.Notice, c.Rules,
-			pq.StringArray(c.Eightball),
+			pq.StringArray(c.Eightball), c.CinemaEnabled,
```

Comment:
This is not extremely important, but these would look better behind eightball.

Reply:
Done, I think.

Reply:
Need to keep same order.

Reply:
Specifically, see
https://github.com/bakape/meguca/pull/1113/commits/9ea786cd947c9a76983f930a91f1b47c1dd0a505#diff-a9739aed10022185142eb131c08fe5efR56
https://github.com/bakape/meguca/pull/1113/commits/9ea786cd947c9a76983f930a91f1b47c1dd0a505#diff-a9739aed10022185142eb131c08fe5efR82
https://github.com/bakape/meguca/pull/1113/commits/9ea786cd947c9a76983f930a91f1b47c1dd0a505#diff-a9739aed10022185142eb131c08fe5efR123

Reply:
Oh, right, now should be good.